Generate.py

```diff
@@ -108,6 +108,7 @@ def main(args=None) -> Tuple[argparse.Namespace, int]:
         meta_weights = None
     player_id = 1
     player_files = {}
+    player_errors = []
```

Comment:
```suggestion
    player_errors: list[str] = []
```
And the line above it isn't something you added, but if you wanted to improve it, you could add `: dict[int, str]`